.github/workflows/test.yml

```diff
@@ -42,11 +31,11 @@ jobs:
       - name: Install package dependencies 📄
         uses: boehringer-ingelheim/dv.templates/.github/actions/dependencies@main     
 
-      - name: Test 📦
+      - name: Test and QC 🧪
         run: Rscript templates/scripts/test.R
```

Comment:
use 
        shell: Rscript {0}
and update path to scripts/test.R (maybe rename also this file to something more explicit ?)

Reply:
does it make sense to run QC unless on PR to main?

Reply:
It now runs using shell.

I would say it makes sense so we have continuous feedback on failing the qc report. It should be no different than running tests.